### ISSUE TITLE
Difference between transformer min and max is critical with regard to creating model infeasabilities

### ISSUE THREAD

**Issue by jokochems** [pommes-public/pommesdispatch#38]
Here is what I have found out so far:
* In the course of introducing the latest `pommesdata` update, I also touched the min and max loads.
* Hereby, I introduced / changed the minimum distance between min and max loads.
* It seems that the model configuration requires the margin to be not too small since otherwise other constraints may not be met and we end up with an infeasible model.
* Since this also influences results quality, I suggest to make it a time-dependent assumption input. I see the following justification:
  * For 2017 as historical year, it is working out fine with rather limited margins. Since we want to replicate historical behaviour, this improves result quality.
  * For 2030 or other future years, you could assume the market-based flexibility of plants increasing (gradually over time). Thus, you can justify higher maximum loads which might be required to fill longer-term "kalte Dunkelflauten".

@yannickwerner You have been working on the minimum and maximum loads part mostly, esp. for the IPP and CHP part. What is your opinion on that?
* Do you see any strong counterargument towards proceeding as I suggested?
* Also, could you think of which constraints being violated if our minimum loads are too low? They are still higher than the maximum load ones. So I don't see any direct and easy explanation, but rather suppose a complicated interference with other model components.

Anyways, I'll also try to dig a little deeper into that.

**Comment by jokochems** [comment]
I've looked into it again and here is what I found out:
* It is a problem with the power plants themselves and happens especially for foreign gas plants with a minimum output. It can occur for other plants with a minimum load time series as well, though. At the moment, they simply have enough ramping capability.
* We derive European power plants' ramping capability from historically observed values. Besides, we define a minimum and maximum load pattern, whereby the maximum load is basically the minimum load plus a small adder as margin.
* Now if both, the gradient and the margin from `min` to `max` are set too restrictive, this may lead to model infeasibilities since the plant might not be able to ramp up / down as much as it would require to without violating minimum loads.

I suggest to do the following two things:
1. Increase the gradient for future years for European gas power plants to depict an increase in flexibility. We limit it mainly due to market, not to technical reasons at the moment.
2. If needed to retain model feasibility, also slightly increase the margin to deviate from the minimum load requirement over time for foreign and maybe even for domestic plants. Again, our limitation is introduced mainly due to empirical market observations rather than any technical requirements.

@yannickwerner To address this, I've opened a new Pull Request in `pommesdata` which I based on the latest version: https://github.com/pommes-public/pommesdata/pull/41
